src/Codeception/PHPUnit/Listener.php

```diff
@@ -10,119 +11,246 @@
 use PHPUnit_Framework_Test;
 use Symfony\Component\EventDispatcher\EventDispatcher;
 
-class Listener implements \PHPUnit_Framework_TestListener
-{
-    /**
-     * @var \Symfony\Component\EventDispatcher\EventDispatcher
-     */
-    protected $dispatcher;
+// PHPUnit 6 definition for the TestListener class includes a new method `addWarning`
+if (interface_exists('PHPUnit\Framework\TestListener')) {
```

Comment:
Yeah, now that I see it a second time, the solution was way simpler. I'll change it,